config/msb/conf.d/imu.yaml

```diff
@@ -1,6 +1,6 @@
 verbose         : false     # print debugging information
 print_stdout    : false     # print data to stdout
-sample_rate_div : 31        # output data rate (odr) is calculated like this: odr = 1125 / (1 + sample_rate_div)
+sample_rate_divisor : 31        # output data rate (odr) is calculated like this: odr = 1125 / (1 + sample_rate_div)
```

Comment:
This is a bug for IMU, should not be fixed here.

Reply:
done